cognite/client/data_classes/contextualization.py

```diff
@@ -607,6 +637,32 @@ class PersonalProtectiveEquipmentDetectionParameters(VisionResource, ThresholdPa
     pass
 
 
+@dataclass
+class DialGaugeDetection(VisionResource, ThresholdParameter):
+    min_level: float | None = None
+    max_level: float | None = None
+    dead_angle: float | None = None
+    non_linear_angle: float | None = None
```

Comment:
Why do we include non linear angle @miladh  ?

(I removed it from the api docs last week https://github.com/cognitedata/service-contracts/pull/2166/commits/868e315c9393b16bb99b269326c666769283ca39 , but it was added again https://github.com/cognitedata/service-contracts/pull/2166/commits/c6dd5ff55c126979e68613e5f3e05c25aff6026a)

Reply:
https://github.com/cognitedata/service-contracts/pull/2166#discussion_r1347324758